templates/All/Default.html

```diff
@@ -1,44 +1,36 @@
 <div class="coveo-result-frame">
-  <div class="coveo-result-row">
-    <div class="coveo-result-cell" style="width:85px;text-align:center;padding-top:7px;">
-      <span class="CoveoIcon">
-      </span>
-      <div class="CoveoQuickview">
+  <div class="coveo-result-cell" style="vertical-align:top;text-align:center;width:32px;">
```

Comment:
I don't know if we want to make that the default "style" or leave people adding in-line style to their own templates.

2 scenarios:

1. If we make css classes, someone would have to inspect the element to find what to override, and possible have to "reset" some properties like margins, etc.

2. If we leave it in-line it's heavier on the code, but easier to change.

The interface editor has controls for these kind of properties:
![image](https://cloud.githubusercontent.com/assets/2488155/26453696/a0061bdc-4132-11e7-91cd-97b593f52ccc.png)

So I chose to leave them there.